packages/@lightningjs/ui-components/src/mixins/withLayout/withLayout.stories.js

```diff
@@ -49,25 +49,25 @@ export const withLayout = () => {
 withLayout.storyName = 'withLayout';
 
 withLayout.args = {
-  ratioX: 16,
+  ratioX: -16,
```

Comment:
should this one be negative or was this just a local test?

Reply:
Crud. Yea that was a local test that got pushed. I'll revert it.